mint.json

```diff
@@ -111,6 +111,34 @@
     {
       "group": "Configurations",
       "pages": [
+        {
+          "icon": "user-lock",
+          "group": "Authentication & Personalization",
```

Comment:
Can we change this to `Auth & Personalization` it looks kinda cramped

Reply:
updated!